### ISSUE TITLE
Evaporation

### ISSUE THREAD

**Issue by bwj202** [dvalters/HAIL-CAESAR#52]
Hi, can you confirm that evaporation isn't currently implemented in HAIL-CAESAR right now?

**Comment by dvalters** [comment]
Hi, it's not currently tested, but the code is there. It wouldn't take too much to finalise the implementation/check as the routines are fairly straightforward.

**Comment by bwj202** [comment]
Yeah that would be really helpful if you could implement it, we're modelling a beaver dam site so there are lots of potential losses. Appreciate the work by the way, I need to get myself away from the world of Python and learn a proper language one of these days.
